backend/LexBoxApi/Services/HgService.cs

```diff
@@ -137,6 +139,8 @@ public async Task FinishReset(string code, Stream zipFile)
         // Now we're ready to move the new repo into place, replacing the old one
         await DeleteRepo(code);
         tempRepo.MoveTo(PrefixRepoFilePath(code));
+        // Force NFS to refresh its directory cache.
+        Process.Start("ls", [PrefixRepoFilePath(code)]);
```

Comment:
should protect this behind an OS check similar to the file permissions in other places

Reply:
Ah, right, because this won't *always* be running inside a Kubernetes pod — some people are running the lexbox API on their Windows dev machines. Well, this isn't the final version; once I'm finally able to prove whether this makes a difference, I'll test it with Directory.GetFiles and if GetFiles works, I'll just use GetFiles as that's more efficient anyway (shelling out to `ls` runs another process, whereas GetFiles just calls the appropriate kernel syscalls).

Reply:
Moot now since I've found another way to do this.